### ISSUE TITLE
 Failed assertion: line 780 pos 12: 'elapsedInSeconds >= 0.0': is not true  package:flutter/src/animation/animation_controller.dart

### ISSUE THREAD

**Issue by AsmaaMohamedHelali** [flutter/flutter#54037]
## Steps to Reproduce

1. Run `flutter create bug`.
2. Update the files as follows: ... 
3. ... 

**Expected results:** 

**Actual results:** 

<details>
  <summary>Logs</summary>

```
```

```
```

```
```

</details>

**Comment by AsmaaMohamedHelali** [comment]
This exception was thrown in the context of a scheduler callback. When the scheduler callback was _registered_ (as opposed to when the exception was thrown), this was the stack: 
#0      new _FrameCallbackEntry.<anonymous closure> (package:flutter/src/scheduler/binding.dart:116:33)
#1      new _FrameCallbackEntry (package:flutter/src/scheduler/binding.dart:119:6)
#2      SchedulerBinding.scheduleFrameCallback (package:flutter/src/scheduler/binding.dart:518:49)
#3      Ticker.scheduleTick (package:flutter/src/scheduler/ticker.dart:252:46)
#4      Ticker.start (package:flutter/src/scheduler/ticker.dart:161:7)

**Comment by TahaTesser** [comment]
Hi @AsmaaMohamedHelali 
can you please provide your `flutter doctor -v` and `flutter run --verbose`? Also, to better address the issue, would be helpful
if you could post a minimal code sample to reproduce the problem
Thank you

**Comment by AsmaaMohamedHelali** [comment]
flutter doctor -v
√] Flutter (Channel stable, v1.12.13+hotfix.8, on Microsoft Windows [Version 10.0.18362.720], locale en-VI)
    • Flutter version 1.12.13+hotfix.8 at C:\Users\20100\Documents\me\flutter_windows_v1.12.13+hotfix.7-stable\flutter
    • Framework revision 0b8abb4724 (8 weeks ago), 2020-02-11 11:44:36 -0800
    • Engine revision e1e6ced81d
    • Dart version 2.7.0


[√] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at C:\Users\20100\AppData\Local\Android\sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.2
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b04)
    • All Android licenses accepted.

[√] Android Studio (version 3.6)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 44.0.2
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b04)

[√] VS Code (version 1.38.1)
    • VS Code at C:\Users\20100\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.5.1

[!] Connected device
    ! No devices available
----------------------------------------------------------
flutter run --verbose
[  +36 ms] executing: [C:\Users\20100\Documents\me\flutter_windows_v1.12.13+hotfix.7-stable\flutter\] git -c log.showSignature=false log
-n 1 --pretty=format:%H
[  +85 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +1 ms] 0b8abb4724aa590dd0f429683339b1e045a1594d
[        ] executing: [C:\Users\20100\Documents\me\flutter_windows_v1.12.13+hotfix.7-stable\flutter\] git describe --match v*.*.*
--first-parent --long --tags
[  +58 ms] Exit code 0 from: git describe --match v*.*.* --first-parent --long --tags
[        ] v1.12.13+hotfix.8-0-g0b8abb472
[  +11 ms] executing: [C:\Users\20100\Documents\me\flutter_windows_v1.12.13+hotfix.7-stable\flutter\] git rev-parse --abbrev-ref
--symbolic @{u}
[  +48 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/stable
[        ] executing: [C:\Users\20100\Documents\me\flutter_windows_v1.12.13+hotfix.7-stable\flutter\] git ls-remote --get-url origin
[  +53 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[ +100 ms] executing: [C:\Users\20100\Documents\me\flutter_windows_v1.12.13+hotfix.7-stable\flutter\] git rev-parse --abbrev-ref HEAD
[  +52 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] stable
[ +156 ms] executing: C:\Users\20100\AppData\Local\Android\sdk\platform-tools\adb.exe devices -l
[  +35 ms] Exit code 0 from: C:\Users\20100\AppData\Local\Android\sdk\platform-tools\adb.exe devices -l
[        ] List of devices attached
[  +15 ms] No supported devices connected.
[  +22 ms] "flutter run" took 272ms.

#0      throwToolExit (package:flutter_tools/src/base/common.dart:28:3)
#1      RunCommand.validateCommand (package:flutter_tools/src/commands/run.dart:289:7)
<asynchronous suspension>
#2      FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:590:11)
#3      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:515:33)
<asynchronous suspension>
#4      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:146:29)
#5      _rootRun (dart:async/zone.dart:1126:13)
#6      _CustomZone.run (dart:async/zone.dart:1023:19)
#7      _runZoned (dart:async/zone.dart:1518:10)
#8      runZoned (dart:async/zone.dart:1465:12)
#9      AppContext.run (package:flutter_tools/src/base/context.dart:145:18)
#10     FlutterCommand.run (package:flutter_tools/src/runner/flutter_command.dart:505:20)
#11     CommandRunner.runCommand (package:args/command_runner.dart:197:27)
#12     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:413:21)
<asynchronous suspension>
#13     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:146:29)
#14     _rootRun (dart:async/zone.dart:1126:13)
#15     _CustomZone.run (dart:async/zone.dart:1023:19)
#16     _runZoned (dart:async/zone.dart:1518:10)
#17     runZoned (dart:async/zone.dart:1465:12)
#18     AppContext.run (package:flutter_tools/src/base/context.dart:145:18)
#19     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:364:19)
#20     CommandRunner.run.<anonymous closure> (package:args/command_runner.dart:112:25)
#21     new Future.sync (dart:async/future.dart:224:31)
#22     CommandRunner.run (package:args/command_runner.dart:112:14)
#23     FlutterCommandRunner.run (package:flutter_tools/src/runner/flutter_command_runner.dart:250:18)
#24     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:63:22)
#25     _rootRun (dart:async/zone.dart:1126:13)
#26     _CustomZone.run (dart:async/zone.dart:1023:19)
#27     _runZoned (dart:async/zone.dart:1518:10)
#28     runZoned (dart:async/zone.dart:1502:12)
#29     run.<anonymous closure> (package:flutter_tools/runner.dart:61:18)
<asynchronous suspension>
#30     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:146:29)
#31     _rootRun (dart:async/zone.dart:1126:13)
#32     _CustomZone.run (dart:async/zone.dart:1023:19)
#33     _runZoned (dart:async/zone.dart:1518:10)
#34     runZoned (dart:async/zone.dart:1465:12)
#35     AppContext.run (package:flutter_tools/src/base/context.dart:145:18)
#36     runInContext (package:flutter_tools/src/context_runner.dart:64:24)
#37     run (package:flutter_tools/runner.dart:50:10)
#38     main (package:flutter_tools/executable.dart:65:9)
#39     main
(file:///C:/Users/20100/Documents/me/flutter_windows_v1.12.13+hotfix.7-stable/flutter/packages/flutter_tools/bin/flutter_tools.dart:8:3)
#40     _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:305:32)
#41     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:174:12)
---------------------------------------------
and about my code i don't know which part of my code causes that error so i can post it  but i thought it causes because SingleTickerProviderStateMixin i am using tabs in my login screen and navigate to the home screen when user successfully login but when i logged out and remove cache navigating back to my login screen that error appears in run concol but nothing happened in my application

**Comment by TahaTesser** [comment]
Hi @AsmaaMohamedHelali 

Please execute `flutter run --verbose` from root of your project
```
[ ] List of devices attached
[ +15 ms] No supported devices connected.
[ +22 ms] "flutter run" took 272ms.
```

Also, to better address the issue, would be helpful if you could post a minimal code sample to reproduce the problem
Thank you

**Comment by khashabawy** [comment]
════════ Exception caught by scheduler library ═════════════════════════════════════════════════════
The following assertion was thrown during a scheduler callback:
'package:flutter/src/animation/animation_controller.dart': Failed assertion: line 780 pos 12: 'elapsedInSeconds >= 0.0': is not true.


Either the assertion indicates an error in the framework itself, or we should provide substantially more information in this error message to help you determine and fix the underlying cause.
In either case, please report this assertion by filing a bug on GitHub:
  https://github.com/flutter/flutter/issues/new?template=BUG.md

When the exception was thrown, this was the stack: 
#2      AnimationController._tick (package:flutter/src/animation/animation_controller.dart:780:12)
#3      Ticker._tick (package:flutter/src/scheduler/ticker.dart:237:5)
#4      SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1102:15)
#5      SchedulerBinding.handleBeginFrame.<anonymous closure> (package:flutter/src/scheduler/binding.dart:1017:11)
#6      _LinkedHashMapMixin.forEach (dart:collection-patch/compact_hash.dart:379:8)
...

This exception was thrown in the context of a scheduler callback. When the scheduler callback was _registered_ (as opposed to when the exception was thrown), this was the stack: 
#0      new _FrameCallbackEntry.<anonymous closure> (package:flutter/src/scheduler/binding.dart:116:33)
#1      new _FrameCallbackEntry (package:flutter/src/scheduler/binding.dart:119:6)
#2      SchedulerBinding.scheduleFrameCallback (package:flutter/src/scheduler/binding.dart:518:49)
#3      Ticker.scheduleTick (package:flutter/src/scheduler/ticker.dart:252:46)
#4      Ticker.start (package:flutter/src/scheduler/ticker.dart:161:7)
...
════════════════════════════════════════════════════════════════════════════════════════════════════

**Comment by TahaTesser** [comment]
Hi @AsmaaMohamedHelali 
Without additional information, we are unfortunately not sure how to resolve this issue. We are
therefore reluctantly going to close this bug for now.

if you're still experiencing this problem, please create a new issue and provide the information
required in the template.
Thank you

**Comment by rivetingpeppermint** [comment]
I have this issue when dismissing dialogs. Here's a snippet of my code:

**Comment by TahaTesser** [comment]
Could everyone who still has this problem please file a new issue with the exact description of what happens, logs and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.

**Comment by mebden** [comment]
I got this many times in a row while trying to close dialog boxes on an iPad device, through the latest Flutter (1.21.0-5.0.pre) and the brand new Xcode 11.6. I couldn’t figure out what started the problem. What didn't work was closing Xcode, running `flutter clean` & `flutter pub get`, reopening Xcode and rebuilding the project.

I switched from Debug mode in the Xcode gui to **Release** mode at the command line, and so far the problem hasn't reappeared. 

**TahaTesser:** I don't want to submit my code so won't file a bug report. Errors etc are in the ASCII file below in case they are still of interest to anyone.

[bug.txt](https://github.com/flutter/flutter/files/5008921/bug.txt)
